pkgs/by-name/la/lammps/package.nix

```diff
@@ -44,9 +45,27 @@
   extraCmakeFlags ? { },
   # Extra `buildInputs` - meant for packages that require more inputs
   extraBuildInputs ? [ ],
+  # If false, the `stdenv` argument will be used no matter what is the state of
+  # extraCmakeFlags.GPU_API or config.cudaSupport . See logic for stdenv' below.
+  overrideStdenv ? true,
```

Comment:
H'm, how do you intend to use this flag?

Reply:
> H'm, how do you intend to use this flag?

Not me, but see my reply to @markuskowa [here](https://github.com/NixOS/nixpkgs/pull/366528#discussion_r1898951393).